pyvona.py

```diff
@@ -126,6 +126,17 @@ def fetch_voice_fp(self, text_to_speak, fp):
         else:
             fp.write(r.content)
 
+    def fetch_voice_data(self, text_to_speak):
+        """Fetch voice data for given text and return data
+        """
```

Comment:
I'd like to see the redundant code between this function and `fetch_voice_fp` removed. Should just be a matter of making `fetch_voice_fp` call `fetch_voice_data` with `self` and `text_to_speak`, and writing the returned value to the filepointer `fp`. Also, as a matter of preference, once `fetch_voice_data` is called by `fetch_voice_fp`, I'd like to see it moved to above `fetch_voice_fp`. Thanks!